org.eclipse.lsp4e/src/org/eclipse/lsp4e/LanguageServiceAccessor.java

```diff
@@ -204,6 +206,27 @@ private static boolean capabilitiesComply(LanguageServerWrapper wrapper,
 				|| capabilitiesPredicate.test(castNonNull(wrapper.getServerCapabilities()));
 	}
 
+	/**
+	 * Asynchronously checks if the given {@code wrapper}'s capabilities comply with the given
+	 * {@code capabilitiesPredicate}.
+	 *
+	 * @param wrapper
+	 *            the server that's capabilities are tested with
+	 *            {@code capabilitiesPredicate}
+	 * @param capabilitiesPredicate
+	 *            predicate testing the capabilities of {@code wrapper}.
+	 * @return The result of applying the capabilities of {@code wrapper} to
+	 *         {@code capabilitiesPredicate}, or {@code false} if
```

Comment:
can you change the javadoc to reflect the change in the PR? Could you also change the javadoc of the method right before, which seems to be wrong as well?